src/xrCore/Containers/AssociativeVector.hpp

```diff
@@ -302,6 +303,11 @@ IC void _associative_vector::insert(TIterator first, TIterator last)
     std::sort(begin(), end(), static_cast<TComparer&>(*this));
 }
 
+TEMPLATE_SPECIALIZATION
+IC typename _associative_vector::insert_result _associative_vector::emplace(key_type key, mapped_type value) {
+    return insert(std::make_pair(key, value));
```

Comment:
It should not call own insert function. It should call emplace from the underlying xr_vector

Reply:
Hey, thanks for the feedback!

Figured that might be the case, although it didn't seem prudent at first considering the vector elements can't ever be anything else than an `std::pair`. I will get back on it, and fix the issue.